Kernel/Arch/init.cpp

```diff
@@ -169,6 +178,22 @@ extern "C" [[noreturn]] UNMAP_AFTER_INIT void init(BootInfo const& boot_info)
     multiboot_framebuffer_height = boot_info.multiboot_framebuffer_height;
     multiboot_framebuffer_bpp = boot_info.multiboot_framebuffer_bpp;
     multiboot_framebuffer_type = boot_info.multiboot_framebuffer_type;
+#elif ARCH(AARCH64)
+    // FIXME: Don't hardcode this and also use the BootInfo structure for the aarch64 build.
```

Comment:
For aarch64 we should use the device tree to probe for things instead of multi boot :) It has a field for reserved memory maps similar to the multi boot memory map for example.

so, better fixme pls

Reply:
Yep, that's true indeed. I changed the FIXME.